packages/core/src/Stores/Constants/markers.js

```diff
@@ -28,6 +28,14 @@ export const MARKER_TYPES_CONFIG = {
         },
         content_config: { line_style: 'solid', label: localize('Start Time') },
     },
+    LINE_RESET: {
+        type: 'LINE_RESET',
+        marker_config: {
+            ContentComponent: MarkerLine,
+            className: 'chart-marker-line',
```

Comment:
I know it's not your code but it's super duper weird to have classnames inside config and add the same classname all over again to each type here. Maybe it's worth moving it out of each type, declare on top and reuse it.

Reply:
I can create a default  `marker_config` object and copy it (by destructuring)